mapping/src/components/Mapping.jsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+const Mapping = () => {
+  return (
+    <ul className="flex flex-col items-center">
+      {["dog", "cat", "chicken", "cow", "sheep", "horse"].map((item, index) => {
```

Comment:
It looks good, however I suggest you to have your array outside the return stored in a variable and just on your mapping part using the variable name.